src/mame/fujitsu/secoinsa20.cpp

```diff
@@ -0,0 +1,626 @@
+{
+}
+
+void secoinsa20_state::secoinsa20(machine_config &config)
+{
+	// basic machine hardware
+	Z80(config, m_maincpu, XTAL(4'000'000));
+}
+
+
+
+ROM_START( secoinsa20 )
+	ROM_REGION(0x20000, "maincpu", 0)
+	ROM_LOAD( "secoinsa_iras_3f_vo2.2_2732a.bin", 0x0000, 0x1000, CRC(50b09416) SHA1(9a799e2e6ebad484226778f41b292a66ebe45849) )
+
+	ROM_REGION(0x10000, "wd_mfm_controller", 0)
+	ROM_LOAD( "62-000019-00-a0_n82s181n.u16",     0x0000, 0x0400, CRC(d7819b5d) SHA1(c74dc7cbe05796ec787db482f4ccc92d93df2fa4) )
+	ROM_LOAD( "62-000019-01-a0_n82s181n.u15",     0x0400, 0x0400, CRC(bb775759) SHA1(dae3fc65e072c5ebec6c545cef4e58ba0dc44ae7) )
+	ROM_LOAD( "62-000019-02-a0_n82s181n.u14",     0x0800, 0x0400, CRC(94f9ed35) SHA1(2a14134849213a733b05b9e216b45909c36bd859) )
+
+	ROM_REGION(0x00117, "plds", 0)
+	ROM_LOAD( "asp1_2m_v00.1_pal16l8.2m",         0x0000, 0x0117, BAD_DUMP CRC(c20eae34) SHA1(9a879350be345ea757ba7b6636a9c789932b75b4) ) // On PCB SM16B-0200-0010, bruteforced
+	ROM_LOAD( "asp2_2l_v00.1_pal16l8.2l",         0x0000, 0x0117, NO_DUMP ) // On PCB SM16B-0200-0010
+	ROM_LOAD( "asp_p3_2k_v00.1_pal16l8.2k",       0x0000, 0x0117, BAD_DUMP CRC(4838b218) SHA1(312a2cca2a403776f6d861763258edee17a040b8) ) // On PCB SM16B-0200-0010, bruteforced
+	ROM_LOAD( "palp1_1d_v00.1_pal16l8.1d",        0x0000, 0x0117, NO_DUMP ) // On PCB SM16B-0200-0170
+	ROM_LOAD( "palp2_1c_v00.1_pal16r4.1c",        0x0000, 0x0117, NO_DUMP ) // On PCB SM16B-0200-0170
+	ROM_LOAD( "palp3_3u_v00.1_pal16l8.3u",        0x0000, 0x0117, NO_DUMP ) // On PCB SM16B-0200-0170
+	ROM_LOAD( "palp4_1b_v00.1_pal16r8.1b",        0x0000, 0x0117, NO_DUMP ) // On PCB SM16B-0200-0170
+	ROM_LOAD( "palp5_2p_v00.2_pal16l8.2p",        0x0000, 0x0117, BAD_DUMP CRC(0a8d42ef) SHA1(5f3af633742fd4859458237bc576671be8f5a898) ) // On PCB SM16B-0200-0170, bruteforced
+	ROM_LOAD( "pcpu_2h_v00.1_pal16l8.2h",         0x0000, 0x0117, NO_DUMP ) // On CPU PCB
+	ROM_LOAD( "pss1_1h_v00.1_pal16l8.1h",         0x0000, 0x0117, NO_DUMP ) // On PCB SM16B-0200-0140
+	ROM_LOAD( "pss2_1k_v00.1_pal16l8.1k",         0x0000, 0x0117, BAD_DUMP CRC(069664b1) SHA1(c5d32b20a3e9985dd1cb23acfe73f1f8be2e270d) ) // On PCB SM16B-0200-0140, bruteforced
+	ROM_LOAD( "pss3_1a_v00.2_pal16r4.1a",         0x0000, 0x0117, NO_DUMP ) // On PCB SM16B-0200-0140
+	ROM_LOAD( "pss4_2g_v00.1_pal16l8.2g",         0x0000, 0x0117, BAD_DUMP CRC(34f23cf6) SHA1(c67f0c1878e1a93787bcd09647162e174218fd6f) ) // On PCB SM16B-0200-0140, bruteforced
+	ROM_LOAD( "pss5_1n_v00.2_pal16r4.1n",         0x0000, 0x0117, NO_DUMP ) // On PCB SM16B-0200-0140
+	ROM_LOAD( "pss6_2m_v00.1_pal16l8.2m",         0x0000, 0x0117, NO_DUMP ) // On PCB SM16B-0200-0140
+ROM_END
+
+} // Anonymous namespace
+
+//    YEAR  NAME        PARENT  COMPAT  MACHINE     INPUT       CLASS             INIT        COMPANY     FULLNAME    FLAGS
+COMP( 1981, secoinsa20, 0,      0,      secoinsa20, secoinsa20, secoinsa20_state, empty_init, "Secoinsa", "Serie 20", MACHINE_NO_SOUND | MACHINE_NOT_WORKING )
```

Comment:
If I'm gathering right this has no sound HW on its own, not even a beeper? Pondering if this is a  `MACHINE_NO_SOUND_HW` case, except for the detail that floppy drives actually do motor sound if enabled ...